wp-cache.php

```diff
@@ -1028,25 +1035,27 @@ function toggleLayer( whichLayer ) {
 				echo '<input class="button-primary" type="submit" name="preload_off" value="' . __( 'Cancel Cache Preload', 'wp-super-cache' ) . '" />';
 			}
 			echo '</div>';
-			wp_nonce_field('wp-cache');
+			wp_nonce_field( 'wp-cache' );
 			echo '</form>';
 		} else {
 			echo '<div class="notice notice-warning"><p>' . __( 'Preloading of cache disabled. Please make sure simple or expert mode is enabled or talk to your host administrator.', 'wp-super-cache' ) . '</p></div>';
 		}
-		break;
+			break;
 		case 'plugins':
-		wpsc_plugins_tab();
-		break;
+			wpsc_plugins_tab();
+			break;
 		case 'debug':
-		wp_cache_debug_settings();
-		break;
+			wp_cache_debug_settings();
+			break;
 		case 'settings':
-		if ( isset( $wp_cache_front_page_checks ) == false )
-			$wp_cache_front_page_checks = true;
-		echo '<form name="wp_manager" action="' . esc_url( add_query_arg( array( 'page' => 'wpsupercache', 'tab' => 'settings' ) ) ) . '" method="post">';
-		wp_nonce_field('wp-cache');
-		echo '<input type="hidden" name="action" value="scupdates" />';
-		?><table class="form-table">
+			if ( isset( $wp_cache_front_page_checks ) == false ) {
+				$wp_cache_front_page_checks = true;
+			}
+			echo '<form name="wp_manager" action="' . esc_url_raw( add_query_arg( 'tab', 'settings', $admin_url ) ) . '" method="post">';
+			wp_nonce_field( 'wp-cache' );
+			echo '<input type="hidden" name="action" value="scupdates" />';
+			?>
+		<table class="form-table">
```

Comment:
I've avoided indentation to reduce number of changes. It'll be fixed in next PRs.